internal/provisioner/plain/controllers/bundle_controller.go

```diff
@@ -392,3 +393,68 @@ func (r *BundleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.ConfigMap{}).
 		Complete(r)
 }
+
+func (r *BundleReconciler) bundleImagePod(pod *corev1.Pod, source rukpakv1alpha1.ImageSource) *corev1.Pod {
```

Comment:
Does this need to be a method on the Bundle controller?

Reply:
we can make it standalone